net/stubby/files/README.md

```diff
@@ -20,10 +20,10 @@ This package has some modifications that makes it differ from the default upstre
 ### General Cleanup
 Comments are removed, etc.
 
-### EDNS Client-Subnet Option Changed to 0
+### EDNS Client-Subnet Option
 The value of "edns_client_subnet_private" is '1' in the upstream default config. This informs the upstream resolver to NOT forward your connection's IP to any other upstream servers. This is good for privacy, but could result in sub-optimal routing to CDNs, etc.
 
-To give a more "comparable" DNS experience similar to google/opendns, this package disables this option.
+We retain the upstream value since privacy is a key user motivation for using DNS-over-TLS, but note users encountering poor routing may consider changing it.
```

Comment:
This type of change should really be done standalone IMO, not buried inside a "fixes and cleanups" type patch.  However, I'm not a user of thisw package

Reply:
I agree. However, this would more closely match upstream. When I originally published this package, I was going with what I believed would be a good overall default config for the largest number of users. However, we could easily argue it should match upstream and users tweak as desired.